css/css-color/oklch-l-almost-1-ref.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<style>
+    .ref { background-color: oklch(1 0.2 45); width: 12em; height: 12em; }
```

Comment:
The test is a bit confusing because with white you don't see anything.
Maybe we should add `border-decoration` for the square and/or edit the wording?

Reply:
There's another test that does the same thing, and I guess I love consistency over quality or something 😅

But both you and @dbaron pointed this out, so I'll fix it, and maybe the existing tests that can be white too...